client/src/components/common/ProfileCard/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+import handymanImg from '../../../assets/handyman.png';
+import Button from '../Button';
+
+const ProfileCard = ({ username, service, country, city, hourRate, bio }) => {
+  return (
+    <div className="card">
+      <div className="card__header">
+        <img className="card__header--img" src={handymanImg} alt="handyman" />
```

Comment:
make the alt dynamic in the image, you can use it as follows:
`${username} profile image`